build.gradle

```diff
@@ -36,8 +37,8 @@ allprojects {
     }
 
     project.ext {
-        compileSdkVersion=27
-        targetSdkVersion=24
+        compileSdkVersion=29
+        targetSdkVersion=29
```

Comment:
Yes, minSdkVersion in next commit will by default be 16 and for flavors r2_9_6 and r2_10_6 will be 24, also java language level for r2_9_6 and r2_10_6 is 1.8 for previous versions it will be 1.7